addon/components/rdf-input-fields/paragraph.js

```diff
@@ -0,0 +1,7 @@
+import Component from '@glimmer/component';
+
+export default class RdfParagraphComponent extends Component {
```

Comment:
FYI if you run into the [no-empty-glimmer-component-classes](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-empty-glimmer-component-classes.md#no-empty-glimmer-component-classes) linting rule you can simply delete the .js file.